acj/static/modules/common/avatar-directive.js

```diff
@@ -6,12 +6,13 @@
 			scope: {
 				userId: '=',
 				avatar: '=',
+                forceDefault: '=?',
 				displayName: '=?',
 				fullName: '=?',
 				me: '=?'
 			},
 			template: '<a ng-href="#/user/{{ userId }}">' +
-						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro" /> ' +
+						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro{{ forceDefault ? \'&f=y\' : \'\' }}" /> ' +
```

Comment:
Static image is OK. The reason I'm suggesting using gravatar still is you don't need to change anything in the frontend.